tests/core/test_verifying.py

```diff
@@ -329,7 +330,7 @@ def test_ecr_missing(seeder):
 
         hby.kevers[hab.pre] = hab.kever
 
-        auth = Authorizer(hby, vdb, eacrdntler.rgy.reger, [LEI1])
+        auth = Authorizer(hby, vdb, eacrdntler.rgy.reger, [LEI1], {"EBA Data Submitter"})
```

Comment:
Would prefer this to use an "EBA Data Submitter" constant here and in the other areas.